src/DependencyInjection/Configuration.php

```diff
@@ -18,8 +18,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('m6_statsd');
+        $treeBuilder = new TreeBuilder('m6_statsd');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
I think we should `require` `"symfony/config": "^4.2"` since this method does not exist before

https://github.com/symfony/config/blob/4.2/Definition/Builder/TreeBuilder.php#L57